src/route-metadata.js

```diff
@@ -55,7 +56,7 @@ const preload = (content, resourcePath, skyPagesConfig) => {
         };
 
         if (preferredName !== undefined) {
-          routeMetadata.name = preferredName;
+          routeMetadata.name = jsonDataUtil.parseAngularBindings(preferredName);
```

Comment:
This isn't going on the same file as the previous json-element-attribute.  Here we are grabbing ALL the html files in order to build our the route-meta data, so it grabs the unparsed files, loops through them and grabs that meta-data.